### ISSUE TITLE
Empty heading tag JS misses content hidden from screen readers

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#954]
https://a11ycheckrules.wpenginepowered.com/wp-admin/post.php?post=1006&action=edit

Examples that should flag:

```
<h1><span aria-hidden="true">heading 1 content</span></h1>
<h2><span aria-hidden="true">heading 2 content</span></h2>
<h3><span aria-hidden="true">heading 3 content</span></h3>
<h4><span aria-hidden="true">heading 4 content</span></h4>
<h5><span aria-hidden="true">heading 5 content</span></h5>
<h6><span aria-hidden="true">heading 6 content</span></h6>
```
